pkg/oci/oci.go

```diff
@@ -331,6 +380,10 @@ func tarFolderToFile(src, filePath string) error {
 			return err
 		}
 
+		header.ModTime = headerTime
```

Comment:
Also, I'll say I don't recall the particulars about `filepath.Walk()` logic. With just a couple of files it may not matter, but could also be a source of non-determinism. That may want a little investigation.

Reply:
Documentation says: 

> The files are walked in lexical order, which makes the output deterministic but requires Walk to read an entire directory into memory before proceeding to walk that directory.

So I think we're fine or are you thinking about something else?